src/modules/cart/repositories/cartRepositories.ts

```diff
@@ -69,6 +72,21 @@ const deleteAllUserCarts = async (userId: string) => {
 const deleteCartById = async (id: string) => {
   await db.Carts.destroy({ where: { id } });
 };
+const findCartByAttributes = async(key1: string, value1:any, key2: string, value2:any): Promise<any> => {
+  return await db.Carts.findOne({ where: { [key1]: value1, [key2]: value2 } })
+}
+const findCartProductsByAttribute = async (value: any) => {
```

Comment:
Function name is different to how it behave,
Call it findCartProductsByCartId. or simply pass in key instead of hardcoding "cartId"